### ISSUE TITLE
Image path does not replaced on Production

### ISSUE THREAD

**Issue by prsolucoes** [prsolucoes/angular2-webpack2-starter#1]
Hi,

Image path is not replaced to "/assets/logo.png" when running from production :(

To test run command:

```
npm install
npm start
```

Now some images to explain better:  

> Image from Dev (command: npm start):  

<img width="308" alt="screenshot 2016-10-11 05 02 33" src="https://cloud.githubusercontent.com/assets/395096/19262962/2d699d20-8f70-11e6-9add-f727f7518b37.png">

> Image from Prod (command: npm webpack-prod):  

<img width="323" alt="screenshot 2016-10-11 05 02 43" src="https://cloud.githubusercontent.com/assets/395096/19262960/2d6294c6-8f70-11e6-89b3-b42a1158f122.png">

**Comment by prsolucoes** [comment]
After check i understand that if i use "/" on image "src" it will concat with "options.htmlLoader.root". So i change "root" directory to check if it will throw error and it happens OK. The problem is only with the "src" attribute that dont change:

![screenshot 2016-10-11 16 43 24](https://cloud.githubusercontent.com/assets/395096/19285968/7d4c9998-8fd2-11e6-8cbe-29a2b3ba1cc4.png)

**Comment by prsolucoes** [comment]
Solved after update deps

**Comment by inzerceubytovani** [comment]
I challenge similar issue now. I replaced ts loader module with @ngtools/webpack. But when I call 
`
const ngtools = require('@ngtools/webpack'); `

Webpack returns me error: 

Error: Cannot find module '@angular/tsc-wrapped/src/tsc'...

I inspect module and I found it has included folder @angular_cli and that has own node_modules where is  that @angular/tsc-wrapped dependency but I don't know why it's not loaded. 

Did you have same problem and find some solution?
